pages/biomarkers/[id].js

```diff
@@ -0,0 +1,115 @@
+// node_modules
+import PropTypes from "prop-types";
+import Link from "next/link";
+// components
+import AliasList from "../../components/alias-list";
+import Attribution from "../../components/attribution";
+import Breadcrumbs from "../../components/breadcrumbs";
+import {
+  DataArea,
+  DataItemLabel,
+  DataItemValue,
+  DataPanel,
+} from "../../components/data-area";
+import { EditableItem } from "../../components/edit";
+import PagePreamble from "../../components/page-preamble";
+// lib
+import buildBreadcrumbs from "../../lib/breadcrumbs";
+import errorObjectToProps from "../../lib/errors";
+import FetchRequest from "../../lib/fetch-request";
+
+export default function Biomarker({
+  biomarker,
+  award = null,
+  lab = null,
+  gene,
+}) {
+  return (
+    <>
+      <Breadcrumbs />
+      <EditableItem item={biomarker}>
+        <PagePreamble />
+        <DataPanel>
+          <DataArea>
+            <DataItemLabel>Name</DataItemLabel>
+            <DataItemValue>{biomarker.name}</DataItemValue>
+            <DataItemLabel>quantification</DataItemLabel>
```

Comment:
The `q` is capitalized in this property’s schema title, so it should be capitalized here.